schema_cache.rst

```diff
@@ -0,0 +1,78 @@
+.. _schema_cache:
+
+Schema Cache
+============
+
+PostgREST uses the database schema cache to get information for several of its features:
+
+- For :ref:`resource_embedding`, it needs relationships' information, such as related tables, foreign keys, cardinalities, etc.

```

Comment:
I think we should be precise here, so no "etc". Also "cardinalities" is more of an internal abstraction that's used in the code, so it shouldn't be included either.

```suggestion
- For :ref:`resource_embedding`, it looks for foreign keys constraints to determine relationships between tables or views.
```

Reply:
Agree, fixed the vague examples for this and the other comments.